src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 The second variant was only marginally faster. Still not anywhere near realtime. I tried using ```h264_nvenc``` instead, but it seems that our version of FFmpeg is compiled without it.

```powershell
C:\repos\Java\DigitalMediaServer\target\bin\win32\ffmpeg64.exe -y -i jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv -bufsize 6
M -maxrate 6M -b:v 4M -c:v libx264 -tune fastdecode -preset superfast -c:a aac -b:a 320k -ar 48000 -pix_fmt yuv420p -f mp4 -movflags frag_keyframe+empty_moov -s
eekable 1 hevc_test1.mp4 -benchmark
ffmpeg version N-80363-g403a53c Copyright (c) 2000-2016 the FFmpeg developers
  built with gcc 5.4.0 (GCC)
  configuration: --enable-gpl --enable-version3 --disable-w32threads --enable-nvenc --enable-avisynth --enable-bzlib --enable-fontconfig --enable-frei0r --enabl
e-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libfreetype --enable-libgme --enable-libgsm --enable-libil
bc --enable-libmodplug --enable-libmfx --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-
librtmp --enable-libschroedinger --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo
-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-l
ibzimg --enable-lzma --enable-decklink --enable-zlib
  libavutil      55. 24.100 / 55. 24.100
  libavcodec     57. 46.100 / 57. 46.100
  libavformat    57. 38.100 / 57. 38.100
  libavdevice    57.  0.101 / 57.  0.101
  libavfilter     6. 46.101 /  6. 46.101
  libswscale      4.  1.100 /  4.  1.100
  libswresample   2.  1.100 /  2.  1.100
  libpostproc    54.  0.100 / 54.  0.100
Input #0, matroska,webm, from 'jellyfish-200-mbps-4k-uhd-hevc-10bit.mkv':
  Metadata:
    COMPATIBLE_BRANDS: iso4hvc1iso6
    MAJOR_BRAND     : iso4
    MINOR_VERSION   : 1
    ENCODER         : Lavf56.3.100
  Duration: 00:00:30.10, start: 0.067000, bitrate: 199827 kb/s
    Stream #0:0(und): Video: hevc (Main 10), yuv420p10le(tv), 3840x2160 [SAR 1:1 DAR 16:9], 29.97 fps, 29.97 tbr, 1k tbn, 29.97 tbc (default)
    Metadata:
      CREATION_TIME   : 2016-02-05 13:43:30
      LANGUAGE        : und
      HANDLER_NAME    : hevc@GPAC0.5.2-DEV-rev565-g71748d7-ab-suite
[libx264 @ 000000000058d460] using SAR=1/1
[libx264 @ 000000000058d460] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 AVX2 LZCNT BMI2
[libx264 @ 000000000058d460] profile High, level 5.1
[libx264 @ 000000000058d460] 264 - core 148 r2694 3b70645 - H.264/MPEG-4 AVC codec - Copyleft 2003-2016 - http://www.videolan.org/x264.html - options: cabac=0 r
ef=1 deblock=0:0:0 analyse=0x3:0x3 me=dia subme=1 psy=1 psy_rd=1.00:0.00 mixed_ref=0 me_range=16 chroma_me=1 trellis=0 8x8dct=1 cqm=0 deadzone=21,11 fast_pskip=
1 chroma_qp_offset=0 threads=18 lookahead_threads=3 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_ad
apt=1 b_bias=0 direct=1 weightb=0 open_gop=0 weightp=0 keyint=250 keyint_min=25 scenecut=40 intra_refresh=0 rc_lookahead=0 rc=abr mbtree=0 bitrate=4000 ratetol=
1.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 vbv_maxrate=6000 vbv_bufsize=6000 nal_hrd=none filler=0 ip_ratio=1.40 pb_ratio=1.30 aq=1:1.00
[mp4 @ 0000000000492680] Using AVStream.codec to pass codec parameters to muxers is deprecated, use AVStream.codecpar instead.
Output #0, mp4, to 'hevc_test1.mp4':
  Metadata:
    COMPATIBLE_BRANDS: iso4hvc1iso6
    MAJOR_BRAND     : iso4
    MINOR_VERSION   : 1
    encoder         : Lavf57.38.100
    Stream #0:0(und): Video: h264 (libx264) ([33][0][0][0] / 0x0021), yuv420p, 3840x2160 [SAR 1:1 DAR 16:9], q=-1--1, 4000 kb/s, 29.97 fps, 11988 tbn, 29.97 tbc
 (default)
    Metadata:
      CREATION_TIME   : 2016-02-05 13:43:30
      LANGUAGE        : und
      HANDLER_NAME    : hevc@GPAC0.5.2-DEV-rev565-g71748d7-ab-suite
      encoder         : Lavc57.46.100 libx264
    Side data:
      cpb: bitrate max/min/avg: 6000000/0/4000000 buffer size: 6000000 vbv_delay: -1
Stream mapping:
  Stream #0:0 -> #0:0 (hevc (native) -> h264 (libx264))
Press [q] to stop, [?] for help
frame=  900 fps= 20 q=-1.0 Lsize=   14134kB time=00:00:29.93 bitrate=3868.4kbits/s speed=0.663x
video:14125kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.059250%
bench: utime=511.839s
bench: maxrss=3117484kB
[libx264 @ 000000000058d460] frame I:4     Avg QP:37.18  size: 75474
[libx264 @ 000000000058d460] frame P:226   Avg QP:40.33  size: 33933
[libx264 @ 000000000058d460] frame B:670   Avg QP:42.06  size:  9691
[libx264 @ 000000000058d460] consecutive B-frames:  0.7%  0.2%  0.0% 99.1%
[libx264 @ 000000000058d460] mb I  I16..4: 61.8% 38.1%  0.1%
[libx264 @ 000000000058d460] mb P  I16..4: 17.4%  8.9%  0.0%  P16..4:  9.6%  0.0%  0.0%  0.0%  0.0%    skip:64.1%
[libx264 @ 000000000058d460] mb B  I16..4:  2.1%  0.7%  0.0%  B16..8:  6.2%  0.0%  0.0%  direct: 1.8%  skip:89.3%  L0:44.9% L1:53.3% BI: 1.8%
[libx264 @ 000000000058d460] final ratefactor: 41.27
[libx264 @ 000000000058d460] 8x8 transform intra:32.0% inter:52.7%
[libx264 @ 000000000058d460] coded y,uvDC,uvAC intra: 3.4% 18.9% 1.2% inter: 0.1% 3.2% 0.0%
[libx264 @ 000000000058d460] i16 v,h,dc,p: 52% 27% 12%  9%
[libx264 @ 000000000058d460] i8 v,h,dc,ddl,ddr,vr,hd,vl,hu: 15% 12% 42% 10%  3%  4%  4%  5%  6%
[libx264 @ 000000000058d460] i4 v,h,dc,ddl,ddr,vr,hd,vl,hu: 39% 19% 32%  6%  1%  1%  1%  1%  1%
[libx264 @ 000000000058d460] i8c dc,h,v,p: 69% 12% 17%  2%
[libx264 @ 000000000058d460] kb/s:3853.06
```